modules/migration-installing-legacy-operator.adoc

```diff
@@ -92,7 +92,13 @@ containers:
 <2> Specify your mirror registry.
 endif::[]
 
+ifdef::installing-3-4,installing-restricted-3-4[]
 . Log in to your {product-title} 3 cluster.
+endif::[]
+
+ifdef::installing-mtc,installing-mtc-restricted[]
+. Log in to your {product-title} source cluster.
```

Comment:
I recommend that you use this wording and remove the conditions. If you say "source cluster", you are covered for all situations.